### DISCUSSION TITLE
Attn: @SimonCozens — Issue with the web version

### DISCUSSION THREAD

**Discussion by LibertyType** [fonttools/fontbakery#4370] [General]
The web version is unusable for me because of the black type on dark blue backgrounds on the List All Checks page. They just aren't readable. Simply setting the type to white would be the ideal fix.

Thanks.

**Comment by davelab6** [comment]
Thanks for reporting this, I've made a PR at #4423 to address this :) 

(In future I recommend creating an Iissue rather than a Discussion on a small-scope topic like this :)
